### ISSUE TITLE
(WEB REPORT BY: orangerush REMOTE: 198.245.63.50:7777) Boo Icon shows up when not dead

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#5801]
8ab9672a79089acba08fe858d79f96781c554d96 
As a grey I if someone uses SE on me I boo shows up. Never happens to any other race in game.

**Comment by Intigracy** [comment]
see also:

#5247
#5112

**Comment by Probe1** [comment]
Can be produced sometimes when removing genetic mutations. I made myself a wizard then toggled every genetic block on, then removed my wizard status.  This removed all my genetic mutations that had verbs and gave me boo.  Will try to find a way to reproduce it.

**Comment by clusterfack** [comment]
No idea whats causing this tbh. Considering its always boo you'd think boo was some form of default spell or something when everything breaks.

**Comment by dylanstrategie** [comment]
Needs rechecking and exact reproduction steps

**Comment by DeityLink** [comment]
So the issue is even stranger than I initially thought.

I reproduced the issue on the main server using @Probe1 's steps. I checked my mob's variables, it had no spells despite having the boo icon, so I checked my client's variables, it had the boo /obj/screen/spell , which I checked, and it had a spellmaster so I checked that one too......and it turned out that the spellmaster's holder was...another ghost on the server, controlled by an entirely different player.

What the heck?
